starter-code/css/main.css

```diff
@@ -8,3 +8,119 @@ body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
 }
+/*
+div {
+  border: 1px solid #17212e;
+}
+*/
+.top {
+  background: linear-gradient(#1c5380, #06365f);/* Standard syntax */
+  background-color: #1c5380;/* For browsers that do not support gradients */
+  background: -webkit-linear-gradient(#1c5380, #06365f);/* For Safari 5.1 to 6.0 */
+  background: -moz-linear-gradient(#1c5380, #06365f);/* For Firefox 3.6 to 15 */
+  width: 100%;
+  height: 350px
+}
+
+.phone-picture {
+  position: fixed;
+  width: 320px;
+  height: 515px;
+  margin-left: 300px;
+  margin-top: 150px;
+}
+
+.logo-login {
+  position: fixed;
+  width: 800px;
+  height: 100px;
+  margin-left: 900px;
```

Comment:
Always avoid using large margin and padding values like this. It will make your visual layout dependent on a single resolution. When you try making the window wider or narrower or viewing the page on a different device, the entire layout breaks. There's always a way to design your page without relying on extreme spacing.